src/wikibots/pas.py

```
@@ -12,10 +13,10 @@
 try:
     sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/lib')
     from lib.bot import BaseBot
-    from lib.wikidata import WikidataProperty
+    from lib.wikidata import WikidataProperty, WikidataEntity
 except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)